stable/kong/values.yaml

```diff
@@ -412,6 +412,11 @@ ingressController:
 
   installCRDs: true
 
+  admissionWebhook:
+    enabled: false
+    failurePolicy: Fail
+    port: 8080
```

Comment:
This might sound strange but any reason for making this configurable and not just hard-coding it?
Do you think there will be users who would want to change this, especially since they will never interact with it directly?

Reply:
I made it configurable because all of the other ports are configurable as well, and if someone was using 8080 for something else and wanted to enable the webhook they'd run into a port collision. I'm fine with hard coding it if you prefer.

Reply:
Let's keep it configurable. It has a sane default and nobody should really notice it.